tests/release/release.go

```diff
@@ -27,6 +28,10 @@ const (
 
 	avgPipelineCompletionTime = 10 * time.Minute
 	defaultInterval           = 100 * time.Millisecond
+
+	failure1ApplicationName       = "test"
```

Comment:
See comments on line 52 about the source namespace var

Reply:
Nafta, pay attention to this comment please. Johnny's suggestion for line 52 was on point. The variable should be more descriptive. Failure 1 means nothing to me.

On top of that, sooner or later you are going to add more failure scenarios. Having `failure1`, `failure2`, `failure3`... it's going to be so messy. On top of that, you will have to rename the namespace variables as you will have new namespaces. You should move all of this to a separate file as rename the variables to simply `applicationName`, `releaseLinkName`, etc.

/cc @kasemAlem